ts/components/services/ServiceListItem.tsx

```diff
@@ -42,16 +53,35 @@ const styles = StyleSheet.create({
   }
 });
 
-export class ServiceListItem extends React.PureComponent<Props> {
+export class ServiceListItem extends React.PureComponent<Props, State> {
   public render() {
+    const { isLongPressModeEnabled } = this.props;
     const potService = this.props.item;
     const enabledChannels = pot.map(potService, service =>
       getEnabledChannelsForService(this.props.profile, service.service_id)
     );
 
-    const onPress = pot.toUndefined(
-      pot.map(potService, service => () => this.props.onSelect(service))
-    );
+    if (pot.isSome(potService)) {
```

Comment:
I suppose you can entirely avoid this block by using **_enabledChannels_**

Reply:
By using enabledChannels

you can change this (**_value_**)
```typescript
<ListItemComponent
    title={serviceName}
    subTitle={inboxEnabledLabel}
    hasBadge={!this.props.isRead}
    onPress={onPress}
    onLongPress={this.props.onLongPress}
    hideSeparator={this.props.hideSeparator}
    style={styles.listItem}
    onSwitch={onSwitch}
    value={this.state.uiEnabledChannels.inbox}
    keySwitch={`switch-service-${profileVersion}`}
    isLongPressModeEnabled={isLongPressModeEnabled}
    />
```

with this (**_value_**)
```typescript
<ListItemComponent
    title={serviceName}
    subTitle={inboxEnabledLabel}
    hasBadge={!this.props.isRead}
    onPress={onPress}
    onLongPress={this.props.onLongPress}
    hideSeparator={this.props.hideSeparator}
    style={styles.listItem}
    onSwitch={onSwitch}
    value={pot.isSome(enabledChannels) && enabledChannels.value.inbox}
    keySwitch={`switch-service-${profileVersion}`}
    isLongPressModeEnabled={isLongPressModeEnabled}
  />
```

Reply:
Ok, I changed the value in ServiceListItem and NewServiceListItem